tests/gocase/unit/hello/hello_test.go

```diff
@@ -44,15 +44,19 @@ func TestHello(t *testing.T) {
 	t.Run("hello with protocol 2", func(t *testing.T) {
 		r := rdb.Do(ctx, "HELLO", "2")
 		rList := r.Val().([]interface{})
-		require.EqualValues(t, rList[2], "proto")
-		require.EqualValues(t, rList[3], 2)
+		require.EqualValues(t, "version", rList[2])
+		require.EqualValues(t, "4.0.0", rList[3])
+		require.EqualValues(t, "proto", rList[4])
+		require.EqualValues(t, 2, rList[5])
 	})
 
 	t.Run("hello with protocol 3", func(t *testing.T) {
 		r := rdb.Do(ctx, "HELLO", "3")
 		rList := r.Val().([]interface{})
-		require.EqualValues(t, rList[2], "proto")
-		require.EqualValues(t, rList[3], 2)
+		require.EqualValues(t, "version", rList[2])
+		require.EqualValues(t, "4.0.0", rList[3])
```

Comment:
Would it be better to use regular expressions here instead of specific versions?
such as `^\d+\.\d+\.\d+$`

Reply:
@jihuayu Yes, but maybe we can update this when we decide to change the Redis compatibility version.